src/encoded/static/components/static-pages/components/TableOfContents.js

```diff
@@ -183,7 +187,7 @@ class TableEntry extends React.Component {
                 (this.props.className ? ' ' + this.props.className : '') +
                 (depth === 0 ? ' top' : '') +
                 (active ? ' active' : '')
-            } data-depth={depth} data-recursion-depth={recurDepth} ref="listItem">
```

Comment:
Was going to refactor these out to be React.createRef() instances but these aren't used/consumed by anything so... removed instead.